awx/ui_next/src/screens/Host/HostAdd/HostAdd.jsx

```diff
@@ -34,9 +42,19 @@ function HostAdd() {
   return (
     <CardBody>
       <HostForm handleSubmit={handleSubmit} handleCancel={handleCancel} />
-      {formError ? <div>error</div> : ''}
+      {formError && (
+        <AlertModal
+          variant="danger"
+          title={i18n._(t`Error!`)}
+          isOpen={formError}
+          onClose={() => setFormError(null)}
+        >
+          {i18n._(t`An error occurred when saving Host`)}
+          <ErrorDetail error={formError} />
+        </AlertModal>
```

Comment:
Per a convo we had w/ @jlmitch5 at a recent sync-up and how we originally wrote this up in [our docs](https://github.com/ansible/awx/blob/devel/awx/ui_next/CONTRIBUTING.md#handling-api-error):

We're really going to want to show non-field errors _on_ the form and not in a modal. The main reason for this is that we have forms in several places that appear in their _own_ modals (prompts, credential plugins, etc.). We don't want to show modal-on-top-of-modal when a non-field api error occurs on these forms.

It doesn't look like the patternfly-react form supports this natively, so if you don't see an obvious way of extending it on our end then we'll need to work across teams.

@jlmitch5 @trahman73

Reply:
I discussed this with @trahman73 last week. His takeaway thoughts were: 
> so if it's an overall form related error, the alert modal applies

Reply:
That's a good point